src/main/java/duke/ByeCommand.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+/**
```

Comment:
```suggestion
package duke;

/**
```

Overall you are very consistent in leaving an empty line between your package/import statements and your Class statement. Just one minor change here to be made!